yast2-kdump-unexpected-issue-20191024.json

```diff
@@ -0,0 +1,16 @@
+{
+  "area": [
+    {
+      "xpos": 211,
+      "ypos": 334,
+      "width": 395,
+      "height": 98,
+      "type": "match",
+      "match": 86
+    }
+  ],
+  "properties": [],

```

Comment:
The green frame of the screenshot is confusing. This needle should be a workaround needle.
Guideline: https://progress.opensuse.org/projects/openqatests/wiki#applying-quotworkaroundquot-needles

Reply:
updated now